site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -1,155 +1,309 @@
-  Then click `Annotate`.
+Supported DL models are suitable only for certain labels.
 
-  ![](/images/image187.jpg)
+To annotate with detectors, do the following:
 
-- Some of models supports attributes annotation (like facial emotions, for example: ``serverless/openvino/omz/intel/face-detection-0205``).
-  In this case you can also match attributes of the DL model with the attributes of a CVAT label.
+1. Click **Magic wand** ![Magic wand](/images/image189.jpg), and go to the **Detectors** tab.
+2. From the **Model** drop-down, select model (see [Detectors Models](#detectors-models)).
+3. From the left drop-down select the DL model label and match it to the right drop-down - the label of your task.
 
-  ![](/images/image187_1.jpg)
+   ![](/images/image187.jpg)
 
-- This action will automatically annotates one frame.
-  In the [Automatic annotation](/docs/manual/advanced/automatic-annotation/) section you can read
-  how to make automatic annotation of all frames.
+4. (Optional) If the model returns masks, and you
+   need to convert masks to polygon, use the **Convert masks to polygons** toggle.
+5. Click **Annotate**.
 
-### Mask RCNN
+Some of the models support attribute annotation like facial emotions,
+for example **Attributed face detection** (`serverless/openvino/omz/intel/face-detection-0205`).
 
-The model generates polygons for each instance of an object in the image.
+In this case, you can also match the attributes of the DL model with the attributes of a CVAT label.
 
-### Faster RCNN
+![](/images/image187_1.jpg)
 
-The model generates bounding boxes for each instance of an object in the image. In this model,
-RPN and Fast R-CNN are combined into a single network.
+This action will automatically annotate one frame.
+In the [Automatic annotation](/docs/manual/advanced/automatic-annotation/) section, you can read
+how to make automatic annotations of all frames.
 
-## Trackers
+### Detectors models
 
-Trackers are used to automatically annotate an object using bounding box.
-Supported DL models are not bound to the label and can be used for any objects.
+<!--lint disable maximum-line-length-->
 
-- Before you start, select the `magic wand` on the controls sidebar and go to the `Trackers` tab.
-  Then select a `Label` and `Tracker` for the object and click `Track`. Then annotate the desired objects with the
-  bounding box in the first frame.
+| Model       | Description                                                                                                                                               |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Mask RCNN   | The model generates polygons for each instance of an object in the image.                                                                                 |
```

Comment:
We definitely supports more detectors. See the full list here: https://github.com/opencv/cvat#deep-learning-serverless-functions-for-automatic-labeling